### DISCUSSION TITLE
ios移动端出现严重bug！

### DISCUSSION THREAD

**Discussion by Lux-X-Over** [obsidianzh/forum#96] [General综合]
如在电脑端更改笔记总目录文件夹的名字(其他文件夹同理)，
同步后，
移动端首先会把总目录下所有笔记及文件夹删除，
再更改笔记总目录文件夹的名字，
最后再把所有资料重新下载回来。

如果改成可以直接变换文件夹名字就好了。

**Comment by Lux-X-Over** [comment]
我的邮箱lux.x.over@qq.com，如有需要可以联系。

**Reply by makerjackie** [reply]
可以到 obsidian 官方库的 issue 下反馈，这里只是中文的讨论区

**Comment by Lux-X-Over** [comment]
感谢提醒



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: jackiexiao ***@***.***&gt;
发送时间: 2021年7月20日 11:47
收件人: obsidianzh/forum ***@***.***&gt;
抄送: Lux-X-Over ***@***.***&gt;, Author ***@***.***&gt;
主题: 回复：[obsidianzh/forum] ios移动端出现严重bug！ (#96)
